packages/server/src/tests/epics/epicArticleCount.ts

```diff
@@ -0,0 +1,89 @@
+import { EpicTest } from '@sdc/shared/dist/types';
+
+const lessThanFiftyTest = '2024-01-22_article_count_copy_less_than_fifty';
+
+export const epicArticleCountCopyLessThanFifty: EpicTest = {
+    name: lessThanFiftyTest,
+    campaignId: lessThanFiftyTest,
+    hasArticleCountInCopy: true,
+    status: 'Live',
+    locations: ['GBPCountries', 'AUDCountries', 'EURCountries', 'UnitedStates', 'International'], // is this right? doesn't map exactly to the spec
```

Comment:
I think this will depend on which regions they want to target and whether they want different copy per region...